app/components/chart-published.tsx

```diff
@@ -53,6 +53,7 @@ import {
   InteractiveFiltersChartProvider,
   InteractiveFiltersProvider,
 } from "@/stores/interactive-filters";
+import { theme } from "@/themes/federal";
```

Comment:
Generally `theme` shouldn't be imported directly, but rather used through:

```ts
import { useTheme } from "@mui/material";

const Component = () => {
  const theme = useTheme();
}
```

see https://v5.mui.com/material-ui/customization/theming/#accessing-the-theme-in-a-component 👀 

With this, in case there's e.g. a black theme in the future, you do not need to edit imports, but rather it would be handled automatically through [ThemeProvider](https://v5.mui.com/material-ui/customization/theming/#theme-provider) 👍